network/dime_message.ksy

```diff
@@ -0,0 +1,61 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  license: CC0-1.0
+  endian: be
+doc: |
+  Direct Internet Message Encapsulation (DIME)
+  is an old Microsoft specification for sending and receiving
+  SOAP messages along with additional attachments,
+  like binary files, XML fragments, and even other
+  SOAP messages, using standard transport protocols like HTTP.
+doc-ref: 
+  - http://xml.coverpages.org/draft-nielsen-dime-02.txt
+  - https://docs.microsoft.com/en-us/archive/msdn-magazine/2002/december/sending-files-attachments-and-soap-messages-via-dime
+seq:
+  - id: dime
+    type: message
+types:
+  message:
+    seq:
+    - id: message
+      type: record
+      repeat: eos
+  record:
+    seq:
+    - id: version
+      type: b5
+    - id: first_record
+      type: b1
+    - id: last_record
+      type: b1
+    - id: chunck_record
+      type: b1
+    - id: type_format
+      type: b4
```

Comment:
Is this an [enum](https://doc.kaitai.io/user_guide.html#enums)?

Reply:
It is kind of an enum. But with options `0x05-0x0F` reserved for future use, but you are right that nobody will probably want those ever indeed. So I made it a enum. Thanks!